tools/.github-cla-signers

```diff
@@ -81,6 +81,7 @@ gilbsgilbs
 glyg
 halfdime-code
 hamalq
+hamistao
```

Comment:
Please sign [the CLA](https://ubuntu.com/legal/contributors).

Otherwise looks good to me!